webpack.config.js

```diff
@@ -6,19 +6,53 @@ const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 const UnminifiedWebpackPlugin = require('unminified-webpack-plugin');
 const ThreadsPlugin = require('threads-plugin');
+const WebpackBeforeBuildPlugin = require('before-build-webpack');
+const fs = require('fs');
 
-module.exports = {
+// the WaitPlugin is copypasta from:
+// https://www.viget.com/articles/run-multiple-webpack-configs-sequentially/
+class WaitPlugin extends WebpackBeforeBuildPlugin {
+  constructor(file, interval = 100, timeout = 10000) {
+    super(function (stats, callback) {
+      let start = Date.now();
+
+      function poll() {
+        if (fs.existsSync(file)) {
+          callback();
+        } else if (Date.now() - start > timeout) {
+          throw Error("Maybe it just wasn't meant to be.");
```

Comment:
A slightly more informative message could be nice :D

Reply:
Haha, done!